README.md

```diff
@@ -147,6 +147,22 @@ mosh-client packages to allow mosh-server usage without Perl.
 Notes for developers
 --------------------
 
+To start contributing to Mosh, install the following dependencies:
+
+```
+$ sudo apt install -y protobuf-compiler libprotobuf-dev libutempter-dev autoconf automake nettle-dev
+$ brew install protobuf automake
```

Comment:
And split this out into a separate macOS instruction?

Reply:
Done